.tekton/integration-tests/pipelines/lightspeed-service-integration-test-pipeline-4.17.yaml

```diff
@@ -0,0 +1,316 @@
+          - name: credentials
+            emptyDir: {}
+        steps:
+          - name: get-kubeconfig
+            ref:
+              resolver: git
+              params:
+                - name: url
+                  value: https://github.com/konflux-ci/build-definitions.git
+                - name: revision
+                  value: main
+                - name: pathInRepo
+                  value: stepactions/eaas-get-ephemeral-cluster-credentials/0.1/eaas-get-ephemeral-cluster-credentials.yaml
+            params:
+              - name: eaasSpaceSecretRef
+                value: $(tasks.eaas-provision-space.results.secretRef)
+              - name: clusterName
+                value: "$(tasks.provision-cluster.results.clusterName)"
+              - name: credentials
+                value: credentials
+          - name: run-e2e-tests
+            resources:
+              requests:
+                cpu: '1'
+                memory: 1Gi
+              limits:
+                memory: 10Gi
+            volumeMounts:
+              - name: azure-openai-token
+                mountPath: /var/run/azure_openai
+              - name: azureopenai-entra-id
+                mountPath: /var/run/azureopenai-entra-id
+              - name: bam-token
+                mountPath: /var/run/bam
+              - name: openai-token
+                mountPath: /var/run/openai
+              - name: watsonx-token
+                mountPath: /var/run/watsonx
+              - name: insights-stage-upload-offline-token
+                mountPath: /var/run/insights-stage-upload-offline-token
+              - name: credentials
+                mountPath: /credentials
+            env:
+              - name: BAM_PROVIDER_KEY_PATH
+                value: "/var/run/bam/token"
+              - name: AZUREOPENAI_PROVIDER_KEY_PATH
+                value: "/var/run/azure_openai/token"
+              - name: OPENAI_PROVIDER_KEY_PATH
+                value: "/var/run/openai/token"
+              - name: WATSONX_PROVIDER_KEY_PATH
+                value: "/var/run/watsonx/token"
+              - name: KUBECONFIG
+                value: "/credentials/$(steps.get-kubeconfig.results.kubeconfig)"
+              - name: OLS_IMAGE
+                value: $(params.serviceimage)
+              - name: ARTIFACT_DIR
+                value: "tests/test_results"
+              - name: SUITE_ID
+                value: "nosuite"
+              - name: KONFLUX_OLS_SERVICE_IMAGE
+                value: $(params.serviceimage)
+            image: registry.access.redhat.com/ubi9/ubi-minimal
+            script: |
+              echo "---------------------------------------------"
+              microdnf -y install git make python3.11 python3.11-devel python3.11-pip shadow-utils tar
+              curl -L -o oc.tar.gz https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/latest-4.16/openshift-client-linux-amd64-rhel9.tar.gz \
```

Comment:
should we use 4.17 client for 4.17 cluster?

Reply:
:o yessss!!! Good catch